src/pal/src/exception/seh.cpp

```diff
@@ -212,6 +212,12 @@ VOID
 SEHProcessException(PEXCEPTION_POINTERS pointers)
 {
     pointers->ContextRecord->ContextFlags |= CONTEXT_EXCEPTION_ACTIVE;
+#if defined(_ARM_) && defined(FEATURE_PAL)
```

Comment:
This `FEATURE_PAL` condition is not needed here because of this code lives in PAL - `FEATURE_PAL` is always defined for it.